packages/dx-react-chart/src/plugins/axis.tsx

```diff
@@ -219,6 +220,7 @@ class RawAxis extends React.PureComponent<RawAxisProps> {
     // Only one of templates is rendered then, the other is discarded.
     return (
       <Plugin name="Axis">
+        <Getter name="axesExist" value={true} />
```

Comment:
It's sufficient to specify just `<Getter name="axesExist" value />` as we do for other boolean getters